### PR TITLE
Add instructions for using 'extra:' fields in landscape.yml

### PR DESCRIPTION
**Pull request by riaankleinhans**
To support the drive for more complete project information in landscape.yml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ Before submitting a new entry it is important to review the following guidelines
 > [!NOTE]
 > At the moment the landscape is generated daily, so once your PR is merged your changes should be visible before 24 hours.
 
+## Extras fields
+
+The `extras` section in the landscape.yml file allows you to include various fields to provide essential details about your project. This section helps users understand the project's purpose and find more information. For a detailed list of possible fields please refer to the [Extras Fields Documentation](./docs/extra_fields.md).
```
Comment (tegioz) [inline README.md:31]: Please note that the section is named `extra` (singular), it appears in plural in a few places in the PR 😇

File: docs/extra_fields.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Instructions for Adding Fields under "extras:"
+
+These fields provide essential information about the item, making it easier for users to understand its purpose and where to find more details.
+The `extras` section in the YAML file allows you to include additional information about your project. You can add various fields to provide essential details, making it easier for users to understand its purpose and where to find more information. You can add various fields such as `description`, `website`, `repo_url`, `logo`, and `category`.
+
+These fields provide essential information about the item, making it easier for users to understand its purpose and where to find more details.
```
Comment (tegioz) [inline docs/extra_fields.md:6]: This line seems to be a duplicate, it appears in the previous paragraph.

File: docs/extra_fields.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+# Instructions for Adding Fields under "extras:"
+
+These fields provide essential information about the item, making it easier for users to understand its purpose and where to find more details.
+The `extras` section in the YAML file allows you to include additional information about your project. You can add various fields to provide essential details, making it easier for users to understand its purpose and where to find more information. You can add various fields such as `description`, `website`, `repo_url`, `logo`, and `category`.
```
Comment (tegioz) [inline docs/extra_fields.md:4]: > You can add various fields such as `description`, `website`, `repo_url`, `logo`, and `category`.

These examples aren't actually fields that belong to the `extra` section, and IIRC there isn't a `website` field.

**Review by tegioz — COMMENTED** [review]
Thanks @riaankleinhans!

I've left some comments 🙂

I'm not really sure if we should be listing all the fields available in the `extra` section, as we already have this information in the [reference documentation in the landscape2 repository](https://github.com/cncf/landscape2/blob/3264c8be7116eb455d2b24e33549004c0f5f4261/docs/config/data.yml#L97-L250). Having them in both places will require maintaining both to keep them up to date.

I thought this would be more like highlighting a few key fields, like the `tag`, and pointing users to the other docs for more details. I also miss the new entry in the PR template that you suggested to encourage users to take a look at this, that was a good idea 🙂

File: README.md (hunk 1 above)
Comment (riaankleinhans) [reply README.md]: Updated to "extra"

File: docs/extra_fields.md (hunk 2 above)
Comment (riaankleinhans) [reply docs/extra_fields.md]: Good catch, but deleted the file

File: docs/extra_fields.md (hunk 3 above)
Comment (riaankleinhans) [reply docs/extra_fields.md]: agree, removed the file and link to you your "extra" fields instructions.

**Comment by riaankleinhans** [comment]
@tegioz I missed the PR adding the "extra" field instructions in the landscape2 repo. I agree, information should live only in one place. deleted the file on this side and linked to yours.

**Pull request merged**